LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-select-element/show-picker-cross-origin-iframe.tentative-expected.txt

```diff
@@ -0,0 +1,10 @@
+CONSOLE MESSAGE: SecurityError: Attempted to use a non-registrable domain.
+
+
+Harness Error (TIMEOUT), message = null
+
+PASS Test showPicker() called from cross-origin iframe
+PASS Test showPicker() called from cross-origin iframe 1
+PASS Test showPicker() called from cross-origin iframe 2
+TIMEOUT Test showPicker() called from cross-origin iframe 3 Test timed out
```

Comment:
Curious what's up with this. Not that it needs to be resolved prior to merging this change.

Reply:
Yeah I'm not really sure the same happens for input showPicker too